### ISSUE TITLE
6.x SQL output

### ISSUE THREAD

**Issue by ShinDarth** [TrinityCore/WowPacketParser#203]
https://github.com/TrinityCore/WowPacketParser/commit/a6d4556f373e7213983270d773faf3487c076f4e

I tried to sniff some creature_template for 6.x,  but it outputs 3.3.5 format instead of 6.x:

```
 INSERT INTO `creature_template` (`entry`, `name`, `femaleName`, `subname`, `IconName`, `type_flags`, `type_flags2`, `type`, `family`, `rank`, `KillCredit1`, `KillCredit2`, `PetSpellDataId`, `modelid1`, `modelid2`, `modelid3`, `modelid4`, `HealthModifier`, `ManaModifier`, `RacialLeader`, `questItem1`, `questItem2`, `questItem3`, `questItem4`, `questItem5`, `questItem6`, `movementId`, `exp_unk`
<ShinDarth> , `VerifiedBuild`) VALUES
```

questItem* fields no longer exist in 6.x database

I even set the database configuration in my WowPacketParser.exe.config

**Comment by Kinzcool** [comment]
It doesn't output 3.3.5 format; it just isn't in synch with TC (see `creature_questitem`).

**Comment by ShinDarth** [comment]
ok, then I think the title of this issue is still valid

**Comment by Carbenium** [comment]
Closed by 3ddbce572bddaf79c3d570cf406cc98d927acbad
